test/opendkim.js

```diff
@@ -119,12 +118,4 @@ exports.hook = {
     test.equals(this.connection.transaction.message_stream.pipe.args[1].line_endings, '\r\n');
     test.done();
   },
-  'once is passed end for the message stream' : function (test) {
-    test.expect(3);
-    this.plugin.verify(Stub(), this.connection);
-    test.ok(this.connection.transaction.message_stream.pipe.called);
-    test.equals(this.connection.transaction.message_stream.once.args[0], 'end');
-    test.isFunction(this.connection.transaction.message_stream.once.args[1]);
-    test.done();
-  },
 };
```

Comment:
I must have forgotten to remove this test when I fixed this bug.  It's almost as though I planned having the exact reason we need continuous integration right in the same commit as the continuous integration.